src/frontend/get-jira-configuration.ts

```diff
@@ -50,6 +50,25 @@ const formatDate = function (date) {
 	};
 };
 
+export const getFailedConnections = (installations, subscriptions) => {
+	return installations
+		.filter((response) => !!response.error)
+		.map((failedConnection) => ({
+			...failedConnection,
+			...subscriptions.find(
+				(sub) =>
+					failedConnection.id === sub.getDataValue("gitHubInstallationId")
+			),
+		}))
+		.map((sub) => {
+			const repos = sub.dataValues.repoSyncState?.repos;
+			const repoId = Object.keys(sub.dataValues.repoSyncState?.repos || {});
+			const orgName = repos[repoId[0]].repository?.owner.login || undefined;
```

Comment:
We decided that this will be addressed in a different PR by calling the github API to get the organization of the installation: https://docs.github.com/en/rest/reference/apps#get-an-organization-installation-for-the-authenticated-app